simple/src/collection/ArrayListUse.java

```diff
@@ -0,0 +1,51 @@
+package collection;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+
+public class ArrayListUse {
+    public static ArrayList arrayDetail() {
+        ArrayList arrayListMethod = new ArrayList(Arrays.asList(1000, 2000, 9999));
+
+        return arrayListMethod;
+    }
+
+    public static void arraylistOnePint() {
```

Comment:
please check for spelling mistakes while creating variable, method and classes

Reply:
done